contrib/auth_delay/meson.build

```diff
@@ -14,4 +14,8 @@ auth_delay = shared_module('auth_delay',
   auth_delay_sources,
   kwargs: contrib_mod_args,
 )
-contrib_targets += auth_delay
```

Comment:
Here and everywhere else: why only add it to `contrib_tarets` when `llvm.found()`?

Reply:
Accidentally removed, fixup: f26737a419783745059b95494dd72e51c309755b